SPECS/xsconsole.spec

```diff
@@ -74,6 +71,16 @@ Console tool for configuring a XCP-ng installation.
 %{_unitdir}/xsconsole.service
 
 %changelog
+* Tue Sep 02 2025 Guillaume Thouvenin <guillaume.thouvenin@vates.tech> - 11.0.9.1-1.1
+- Sync with 11.0.9.1-1
+- Drop xsconsole-11.0.8-sync-network-reset-trigger-file.patch
+- Drop xsconsole-11.0.8-do-not-truncate-ipv6-in-pool-config-on-reset.patch
```

Comment:
Have you verified that these are really present in version 11.0.9.1? I see no mention of these changes in XS's changelog.

Reply:
That's master, the tarball does include it.